### PR TITLE
add unit tests

### PR DESCRIPTION
**Pull request by arthosofteq**

### CODE REVIEW

**Review by AmirAllayarovSofteq — APPROVED** [review]
LGTM

File: redisinsight/api/config/test.ts [hunk 1]
```diff
@@ -3,4 +3,7 @@ export default {
     env: 'test',
     requestTimeout: 1000,
   },
+  profiler: {
+    logFileIdleThreshold: parseInt(process.env.PROFILER_LOG_FILE_IDLE_THRESHOLD, 10) || 1000 * 3, // 3sec
```
Comment (AmirAllayarovSofteq) [inline redisinsight/api/config/test.ts:7]: Probably better move it to constant, like DEFAULT_LOG_FILE_THRESHOLD

Comment (arthosofteq) [reply redisinsight/api/config/test.ts:7]: To move what? We are already in config file.

**Review by AmirAllayarovSofteq — APPROVED** [review]
LGTM

**Pull request merged**